build_tools/cmake/iree_llvm.cmake

```diff
@@ -153,6 +153,8 @@ macro(iree_llvm_set_bundled_cmake_options)
 
   # Unconditionally enable some other cheap LLVM tooling.
   set(IREE_LLVM_LINK_TARGET llvm-link)
+  set(IREE_FILECHECK_TARGET FileCheck)
+  set(IREE_NOT_TARGET not)
```

Comment:
I'm not sure if this `IREE_*_TARGET` code covers all the cases we might care about, but it seems easy enough to do more plumbing as needed. `IREE_CLANG_TARGET` in particular has lots of special handling.